### PR TITLE
Story book test

### PR DESCRIPTION
**Pull request by TimGraf**
This does not have to be merged.  I just wanted to get some eyes on it before I get too far down this path.

I updated the README for Storybooks and running them if you are interested in pulling this branch and checking it out.

### CODE REVIEW

**Comment by TimGraf** [comment]
So this is the first pass.  Basically, I took my approach from the development container and just reproduced it in a storybook.  It's not exactly idiomatic.  I am currently watching the following video that David linked with the Jira story.

https://www.youtube.com/watch?v=PF0Vi-iIyoo&feature=youtu.be
https://jira.attainia.com/browse/PLAT-72

I'll take another pass at including more features from storybook.

**Comment by TimGraf** [comment]
OK, I added more data table stories.  Trying to get this to be more storybook idiomatic.

**Comment by TimGraf** [comment]
I also removed redux since we are just working with the component in the story.

**Comment by TimGraf** [comment]
This video was linked in the Jira story by David but is also well worth checking out.

https://www.youtube.com/watch?v=UxbQ-cGnoCE&feature=youtu.be

**Comment by TimGraf** [comment]
Just an observation, in both of those YouTube videos, the presenters were using Visual Studio Code.  I'm just saying.

VSCode

**Comment by TimGraf** [comment]
I added the info add-on but the prop types are displaying as "other" for the type.  I did see a similar issue on a previous release.  This was supposed to be fixed for this add-on.

https://github.com/storybooks/storybook/issues/954

File: .stories/data-table/dataTableStories.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    .add('Check column, no sort, first row selected',
+        withInfo('A DataTable with a checkbox column and no sorting data for any column, and the check and name columns are fixed.  Also, the first row is selected')(() =>
+            <DataTable
+                rowHeight={50}
+                tableWidth={1253}
+                tableHeight={500}
+                headerHeight={50}
+                hasCheckColumn={true}
+                getNextPage={action('nextPage')}
+                rowSelected={action('selectRow')}
```
Comment (arizonatribe) [inline .stories/data-table/dataTableStories.js]: This looks like an awesome add-on and seems to make redux blend into storybooks much better than I was anticipating they would

**Comment by TimGraf** [comment]
Closing this merge request since I have another branch with more refactoring based on feedback.

**Pull request closed without merging**